pypeit/wavecalib.py

```diff
@@ -208,25 +207,28 @@ def build_wv_calib(self, arccen, method, skip_QA=False):
             arcfitter = autoid.HolyGrail(arccen, par=self.par, ok_mask=ok_mask)
             patt_dict, final_fit = arcfitter.get_results()
         elif method == 'identify':
+            final_fit = {}
             # Manually identify lines
             msgs.info("Initializing the wavelength calibration tool")
-            # Todo : Generalise to multislit case
-            arcfitter = gui_identify.initialise(arccen, par=self.par)
-            final_fit = arcfitter.get_results()
-            slit = 0
-            if final_fit[str(slit)] is not None:
-                ans = 'y'
-                # ans = ''
-                # while ans != 'y' and ans != 'n':
-                #     ans = input("Would you like to store this wavelength solution in the archive? (y/n): ")
-                if ans == 'y' and final_fit[str(slit)]['rms'] < 0.1:
-                    # Store the results in the user reid arxiv
-                    specname = self.spectrograph.spectrograph
-                    gratname = "UNKNOWN"  # input("Please input the grating name: ")
-                    dispangl = "UNKNOWN"  # input("Please input the dispersion angle: ")
-                    templates.pypeit_identify_record(final_fit[str(slit)], self.binspectral, specname, gratname, dispangl)
-                    msgs.info("Your wavelength solution has been stored")
-                    msgs.info("Please consider sending your solution to the PYPEIT team!")
+            # TODO: Move this loop to the GUI initalise method
+            embed()
```

Comment:
Should this be here?

Reply:
The method is not properly implemented and shits the bed for multislit data. So ask whoever approved that PR if it should be here;)